lib/backend/k8s/syncer.go

```diff
@@ -274,7 +275,7 @@ func (syn *kubeSyncer) readFromKubernetesAPI() {
 			log.Debugf("Incoming NetworkPolicy watch event. Type=%s", event.Type)
 			if needsResync = syn.eventTriggersResync(event); needsResync {
 				// We need to resync.  Break out into the sync loop.
-				log.Warn("Event triggered resync: %+v", event)
+				log.Warnf("Event triggered resync: %+v", event)
```

Comment:
I'm not sure - but always doing the formatting when we're not logging doesn't feel right to me.  I guess it won't add much overhead so I'd be ok with merging this.

Would it be better though to add a String() method to the KVPair and then use WithField()